components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/AccountEvent.kt

```diff
@@ -16,6 +16,8 @@ val <T> T.exhaustive: T
 sealed class AccountEvent {
     // A tab with all its history entries (back button).
     class TabReceived(val from: Device?, val entries: Array<TabHistoryEntry>) : AccountEvent()
+    // A notification that the profile has been updated.
+    class ProfileUpdated() : AccountEvent()
```

Comment:
@grigoryk would you like `ProfileUpdated` to contain the updated profile data or should it just stay as a hint that you should fetch the profile?